Data-Science-Brigade/modelo-epidemiologico-sc
# Feature/ch11606/pedro agregação semanal

Changes:
* Added an option (through `-y` or `--weekly` to aggregate data weekly)
  * This aggregation is done in the pre-processing phase
* When loading a model check if both the current and previous models are "weekly"
* When saving a model's data, write a file to indicate that it is weekly
* Changed plotting functions to have a `model_is_weekly` parameter indicating whether the model is weekly or not.

What to test:
* Test if you can train both a weekly and a daily model
* Check if the plotting functions work
* Check if you can load a previous model
* Check if the program fails if you load a previous model that is on a different timescale than your own
